data/links.yaml

```
@@ -1614,7 +1614,7 @@ links:
     submenu:
 
   - &enterprise_tools
-    label: "Enterprise Tools"
+    label: "Subscription Manager"
     url: "https://kat.ft.com/"
```

Can you please update the link as well?

```suggestion
    url: "https://subscription-manager.ft.com/"
```